### ISSUE TITLE
Post MVP 530 EZ static page updates

### ISSUE THREAD

**Issue by aliyahblackmore** [department-of-veterans-affairs/va.gov-team#88787]
# CAIA Follow-up Task Ticket
> [!WARNING]
> Do not use this ticket unless directed to do so by the CAIA team.

## Issue description
Leading up to the 530 EZ MVP launch, content identified that the online form didn't include the details about fully developed claims and submission of medical records if claiming a burial allowance for a service-connected death. 

This was noted and the team determined they'd address this post MVP. These form updates require a few static page updates.

## About your team

- **Team name**: benefits-non-disability
- **Product name:** Application for Burial Benefits (VA Form 530 EZ)
- **Product manager:**  Laura Steele 
- **Link to original CAIA intake related to this work:** 

## Follow-up request
The product team is updating their for to include information about the option to submit medical records or an authorization if you're claiming a burial allowance for a service-connected death. 

We need to update the [static page](https://www.va.gov/burials-memorials/veterans-burial-allowance/) to include information about submitting these documents. The form intro page will link out to this page under H2 "What documents do I need to submit with my application?" 

## Will you release this new product incrementally (for example, release to 25% of users to start)?
- [ ] Yes
- [ ] No

**Note** If you checked yes: Incremental launches require dynamic content. You'll need to ask your developer to build a React widget and provide CAIA with the ID code. CAIA will provide widget content and add the widget to your product’s static page. Please refer to this GitHub page for more information.

- [ ] I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as these additional steps will impact deadlines.

**Comment by aliyahblackmore** [comment]
@megzehn here's a ticket to track the other burial allowance static page updates! 

- We only updated the information under "What documents do I need to submit with my application?" to align with the form updates. We can plan to revisit this section of the page in the future. 

- The Drupal version that includes this content is labeled: 
_**7/18/2024 -6:23pm by aliyah.blackmore@va.gov 
Adding one sentences from SME approval to supporting documents section - this is ready to publish after the new H1 changes.**_

I'm still waiting for the product team to confirm if these updates will happen after the H1 change. If these updates are first, we'd just need to adjust the H1 and online form link to match what's on the current live version of the page.

fysa @aprocik1

**Comment by aliyahblackmore** [comment]
Hi @megzehn 

The product team launched the form updates, so we're ready to publish the corresponding static page updates. These were already reviewed and approved by their SMEs. 

**Here's the preview link:**
https://preview-prod.vfs.va.gov/preview?nodeId=864

**Here's info on the adjustments:** 
- We adjusted the static page H1 to align with the product teams adjustment to the breadcrumb, form URL link, and intro page H1. The new H1 is "Veterans burial allowance and transportation benefits". 
- And we adjusted the content under the H1 "What documents do I need to submit with my application?" to align with form updates re: applicants who submit a burial allowance for a service-connected death. They can (optionally) submit medical records or submit authorization for VA to retrieve those records with their form.
- And we only updated the information under "What documents do I need to submit with my application?" to align with the form updates. We can plan to revisit this section and other areas on this page in the future. 

fysa @aprocik1

**Comment by megzehn** [comment]
Confirming I published these changes last week. I think we're good to close this. Does that sound right, @aliyahblackmore?

**Comment by aliyahblackmore** [comment]
Yes @megzehn! Thank you so much for publishing. I'm going to move this to "Validate".
